pyoozie/tags.py

```diff
@@ -69,19 +74,21 @@ class XMLSerializable(object):
     __metaclass__ = abc.ABCMeta
 
     def __init__(self, xml_tag):
+        # type: (typing.Text) -> None
         self.xml_tag = xml_tag
 
     def xml(self, indent=False):
+        # type: (bool) -> str
```

Comment:
Since we define the doc as being utf8, let's return a byte string and explicitly encode as utf-8.